src/devices/tuya.ts

```diff
@@ -8712,6 +8701,420 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0726', ['_TZ3000_kt6xxa4o']),
+        model: 'QAT42Z3',
+        vendor: 'QA',
+        description: 'QA 3CH Scene Switch',
+        whiteLabel: [{vendor: 'QA', model: 'QAT42ZT3'}],
```

Comment:
If the `whiteLabel.model` is the same as the `model`, you can remove `whiteLabel` entirely